samplers/probability/consistent/statistical_test.go

```diff
@@ -150,7 +150,7 @@ func TestSamplerStatistics(t *testing.T) {
 
 				failed := countFailures(rand.NewSource(seed))
 
-				if failed != 1 && test.seedIndex < 0 {
+				if failed != 1 && test.seedIndex < 0 { //nolint:gocritic
```

Comment:
Can you add a reason for the `nolint`?

e.g.


```suggestion
				if failed != 1 && test.seedIndex < 0 { //nolint:gocritic   // [some reason here]
```